sdk/webpubsub/Azure.Messaging.WebPubSub/src/swagger/WebPubSub.json

```diff
@@ -99,7 +99,7 @@
                     "webpubsub"
                 ],
                 "summary": "Check if the connection with the given connectionId exists.",
-                "operationId": "WebPubSubService_ConnectionExists",
+                "operationId": "WebPubSubService_ConnectionExistsImpl",
```

Comment:
We should be able to do this via a transform in the auto-rest MD (so you don't have to make this change in the swagger itself, so we can keep it in sync with the authoritative version.)

Maybe something like this in the autorest.md?


### Make WebPubSubPermission a regular enum
``` yaml
- from: swagger-document
  where: $.paths
  transform: >
    $["/api/hubs/{hub}/connections/{connectionId}"].head.operationId = "WebPubSubService_ConnectionExistsImpl";
    $["/api/hubs/{hub}/groups/{group}"].head.operationId = "WebPubSubService_GroupExistsImpl";
    $["/api/hubs/{hub}/users/{userId}"].head.operationId = "WebPubSubService_UserExistsImpl";
```

Reply:
Yeah, this is temporary. Having said that, this swagger is already forked (PR a week ago). It's forked to generate some methods as internal (and possibly some other small changes). We should work on getting all these customizations out and get back to the original swagger.